torch/csrc/jit/testing/file_check.cpp

```diff
@@ -111,18 +111,19 @@ size_t assertFind(
 }
 
 size_t assertFind(
-    const std::shared_ptr<std::string>& file,
+    const std::shared_ptr<std::string>& source,
     const std::string& sub,
     size_t start,
     const Check& check) {
-  return assertFind(SourceRange(file, start, file->size()), sub, check);
+  auto file = std::make_shared<File>(source, "<filecheck>", 0);
```

Comment:
This is ugly, file rather than string source needs to made more pervasive here and we need to get rid of the std::shared_ptr<string> inside File.